chatgpt-shell.el

```diff
@@ -119,6 +119,11 @@ https://platform.openai.com/docs/models/model-endpoint-compatibility."
   :type 'string
   :group 'chatgpt-shell)
 
+(defcustom chatgpt-shell-dall-e-size "1024x1024"
```

Comment:
How about nil by default and only add to request if set? Check how `chatgpt-shell-model-temperature` does it.

Reply:
Yup, makes sense. I changed it, although that meant I had to remove the optional `size` parameter from `chatgpt-shell-post-dall-e-prompt`, mostly because I wasn't sure how to incorporate it without complicating the code too much. I think that's fine (overriding the custom var is easy enough), but if you have any tips, I'd be willing to give it a go!

Reply:
Re-added in https://github.com/xenodium/chatgpt-shell/commit/3ec935a72cb156b18e9a3a72b6029e787d34a701